readthedocs/rtd_tests/fixtures/spec/v2/schema.json

```diff
@@ -0,0 +1,288 @@
+          },
+          "default": []
+        }
+      }
+    },
+    "python": {
+      "title": "Python",
+      "description": "Configuration of the Python environment to be used.",
+      "type": "object",
+      "properties": {
+        "version": {
+          "title": "Version",
+          "description": "The Python version (this depends on the build image).",
+          "type": "string",
+          "enum": [
+            "2",
+            "2.7",
+            "3",
+            "3.3",
+            "3.4",
+            "3.5",
+            "3.6"
+          ],
+          "default": "3"
+        },
+        "system_packages": {
+          "title": "System Packages",
+          "description": "Give the virtual environment access to the global site-packages directory.",
+          "type": "boolean",
+          "default": false
+        },
+        "install": {
+          "title": "Install",
+          "description": "Installation of packages and requiremens.",
+          "type": "array",
+          "items": {
+            "anyOf": [
+              {
+                "properties": {
+                  "requirements": {
+                    "title": "Requirements",
+                    "description": "The path to the requirements file from the root of the project.",
+                    "type": "string"
+                  }
+                },
+                "required": [
+                  "requirements"
+                ]
+              },
+              {
+                "properties": {
+                  "path": {
+                    "title": "Path",
+                    "description": "The path to the project to be installed",
+                    "type": "string"
+                  },
+                  "method": {
+                    "title": "Method",
+                    "description": "Install using python setup.py install or pip.",
+                    "enum": [
+                      "pip",
+                      "setuptools"
+                    ],
+                    "default": "pip"
+                  },
+                  "extra_requirements": {
+                    "title": "Extra Requirements",
+                    "description": "Extra requirements sections to install in addition to the package dependencies.",
+                    "type": "array",
+                    "items": {
+                      "type": "string"
+                    },
+                    "default": []
+                  }
+                },
+                "required": [
+                  "path"
+                ]
+              }
+            ]
+          }
+        }
+      }
+    },
+    "sphinx": {
+      "title": "Sphix",
+      "description": "Configuration for sphinx documentation.",
+      "type": "object",
+      "properties": {
+        "builder": {
+          "title": "Builder",
+          "description": "The builder type for the sphinx documentation.",
+          "enum": [
+            "html",
+            "htmldir",
```

Comment:
This option is deprecated in favor of `dirhtml`, maybe we could omit it from here?